srcpkgs/pango/template

```diff
@@ -1,8 +1,7 @@
 # Template file for 'pango'
 pkgname=pango
-reverts="1.44.5_1"
```

Comment:
Please leave it, reverts should be removed only when updating to even greater version.